paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -844,6 +845,7 @@ internal class CustomerSheetViewModel(
                 handleViewAction(CustomerSheetViewAction.OnFormError(error))
             },
             setAsDefaultPaymentMethodEnabled = false,
+            financialConnectionsMode = GetFinancialConnectionsMode(elementsSession = null),
```

Comment:
Do we have no way of activating the killswitch for CustomerSheet? The initial state should be derived from an `ElementsSession`, so we could get that information.

Reply:
Doesn't seem like the session is easily accessible here - I'll create a task and handle it separately -> https://jira.corp.stripe.com/browse/LINK_MOBILE-69